src/test/java/org/itsallcode/openfasttrace/maven/TraceMojoIT.java

```diff
@@ -209,6 +213,32 @@ void testTracingSelectedArtifactTypes() throws Exception
                 equalTo("ok - 2 total\n"));
     }
 
+    @ParameterizedTest(name = "wanted tags {0} finds {1} items")
+    @CsvSource(delimiter = ';', nullValues = "NULL", value =
+    { "NULL; 3", "tagA; 1", "tagB; 1", "tagA,tagB; 2", "tagA,tagB,_; 3", "tagC; 0" })
```

Comment:
What about:
```suggestion
    { "NULL; 3", "tagA; 1", "tagB; 1", "tagA,tagB; 2", "tagA,tagB,_; 3", "tagC; 0", "_; 1" })
```
a test to filter all entities without tag, right?

Reply:
or maybe also 
```suggestion
    { "NULL; 3", "tagA; 1", "tagB; 1", "tagA,tagB; 2", "tagA,tagB,_; 3", "tagC; 0", "_; 1", "_,tagA; 2" })
```